### PR TITLE
ext user search controller test

### PR DESCRIPTION
**Pull request by yingjieMiao**
@eishay

### CODE REVIEW

File: kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala [hunk 1]
```diff
@@ -21,26 +21,75 @@ import scala.concurrent.Await
...
+import com.keepit.search.index.{VolatileIndexDirectoryImpl, IndexDirectory, DefaultAnalyzer}
+import com.keepit.shoebox.ShoeboxServiceClient
+import com.keepit.search.user.UserIndexer
+import com.keepit.search.user.{UserSearchFilterFactory, UserQueryParser}
+import com.keepit.common.healthcheck.AirbrakeNotifier
+
+
 class ExtUserSearchControllerTest extends Specification with SearchApplicationInjector {
 
+  private def setup(implicit client: FakeShoeboxServiceClientImpl) = {
```
Comment (eishay) [inline kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala]: why is client an implicit?

File: kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala [hunk 2]
```diff
@@ -48,11 +97,14 @@ class ExtUserSearchControllerTest extends Specification with SearchApplicationIn
 
         val expected = Json.parse(s"""
           {
-            "hits":[],
-            "context":"AgABAAEA"
-          }
+            "hits":
+              [
+                {"userId":5,"basicUser":{"id":"4e5f7b8c-951b-4497-8661-a1001885b2ec","firstName":"Woody","lastName":"Allen","pictureName":"fake.jpg"},"isFriend":false}
```
Comment (eishay) [inline kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala]: would be more readable if you could break this line to something that could be regarded as a documentation of the output json:

``` json
 {
  "userId":5,
  "basicUser":
    {
      "id":"4e5f7b8c-951b-4497-8661-a1001885b2ec",
      "firstName":"Woody",
      "lastName":"Allen",
      "pictureName":"fake.jpg"
    },
    "isFriend":false
  }
```

Comment (yingjieMiao) [inline kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala]: agree

File: kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/search/test/com/keepit/controllers/ext/ExtUserSearchControllerTest.scala]: it's not needed. removing it.

**Pull request merged**